### PR TITLE
content fixes for documents page

### PR DESCRIPTION
**Pull request by BEISKRAM**

### CODE REVIEW

**Review by lorenzoAtBEIS — DISMISSED** [review]
Approving

File: INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadCostAndPerformanceFigures.cshtml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <p class="govuk-body govuk-!-font-weight-bold">You need to upload evidence to support the claims you've made in <span class="govuk-colour--blueunderline">step 5.7.</span></p>
 
-<p class="govuk-body">You can download a <a href="#" class="govuk-link govuk-!-font-weight-regular">spreadsheet template</a> to show the timeline of cost and performance improvements your technology is likely to have.</p>
\ No newline at end of file
+<p class="govuk-body">You can download a <a href="../media/EEF9 Q5.8. Cost and performance pathway template.xlsx" download class="govuk-link govuk-!-font-weight-regular">spreadsheet template</a> to show the timeline of cost and performance improvements your technology is likely to have.</p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadCostAndPerformanceFigures.cshtml:3]: I recommend using tilde (root operator).

Example : href="~/media/EEF9 Q5.8. Cost and performance pathway template.xlsx"

File: INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadRiskRegister.cshtml [hunk 2]
```diff
@@ -10,5 +10,5 @@
               </ul>
 
 
-              <p class="govuk-body">You can <a href="#" class="govuk-link govuk-!-font-weight-regular">download a risk register template</a> or use your own.</p>
+              <p class="govuk-body">You can <a href="../media/EEF9 Q5.17. Risk register template.xls" download class="govuk-link govuk-!-font-weight-regular">download a risk register template</a> or use your own.</p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadRiskRegister.cshtml:13]: I recommend using tilde (root operator).

Example : href="~/media/EEF9 Q5.8. Cost and performance pathway template.xlsx"

File: INZFS.MVC/Views/QuestionDescription/FinanceProposal/_ProjectCostBreakdown.cshtml [hunk 3]
```diff
@@ -7,5 +7,5 @@
 </div>
 
 <p class="govuk-body-m gov-!-margin-bottom-6">
-    You need to upload a detailed project cost breakdown for your project. Use the <a class="govuk-link" href="#">form provided</a> to show your project's expected costs and how you will use funding.
+    You need to upload a detailed project cost breakdown for your project. Use the <a class="govuk-link" href="../media/EEF9 Q6.2. Project cost breakdown template.xlsx" download>form provided</a> to show your project's expected costs and how you will use funding.
```
Comment (sunilbeis) [inline INZFS.MVC/Views/QuestionDescription/FinanceProposal/_ProjectCostBreakdown.cshtml:10]: I recommend using tilde (root operator).

File: INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadRiskRegister.cshtml (hunk 2 above)
Comment (BEISKRAM) [reply INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadRiskRegister.cshtml]: resolved please review

File: INZFS.MVC/Views/QuestionDescription/FinanceProposal/_ProjectCostBreakdown.cshtml (hunk 3 above)
Comment (BEISKRAM) [reply INZFS.MVC/Views/QuestionDescription/FinanceProposal/_ProjectCostBreakdown.cshtml]: resolved please review

**Review by lorenzoAtBEIS — APPROVED** [review]
Looks good to me. Approved

**Review by sunilbeis — APPROVED** [review]

**Pull request merged**